product_template_multi_link/tests/test_product_template_linker.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2020 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo.tests.common import TransactionCase
+
+
+class TestProductTemplateLinker(TransactionCase):
```

Comment:
You can inherit from `BaseCommon` class and remove the `tracking_disable=True` key from context set in `setUpClass` method

Reply:
I updated it. Thanks :pray: